Makefile

```diff
@@ -102,7 +103,7 @@ deps-test:
 
 # Build docker image
 docker: repo/tesseract repo/tesserocr
-	docker build \
+	$(DOCKER) build \
```

Comment:
It would require changes in a lot of places to follow through with this.

Wouldn't it be easier to `alias docker=podman` on systems where one wants to replace one with the other?

Reply:
That would not work. Aliases are not inherited by child processes.

Reply:
> That would not work. Aliases are not inherited by child processes.

...or shell functions defined in .profile, or symlinking into PATH...

Reply:
I have been using that pattern in our local deployments, to test locally with podman and build/deploy with docker. I could create similar PR while I clean up my local build branches.

>> Aliases are not inherited by child processes.
> ...or shell functions defined in .profile, or symlinking into PATH...

I usually do

```
echo '#!/usr/bin/env bash' > ~/bin/docker
echo 'podman "$@"' > ~/bin/docker
chmod a+x ~/bin/docker
```

but that gets confusing when both docker and podman are installed.

Reply:
sigh, yes, that's a contingency one cannot rule out, I guess. So another round of module updates, it is...